### PR TITLE
Changed dropdown to be closer to the designs

### LINKED ISSUE #1214: Change sort by dropdown
screen shot 2018-10-02 at 10 28 00 am

### PR DESCRIPTION
**Pull request by smcmurtry**
Closes #1214

Still need to change the arrow to be like the designs. I will do that in a separate PR.

### CODE REVIEW

File: theme.js [hunk 1]
```diff
@@ -32,7 +34,8 @@ const theme = {
     tornadoRed: "#ea4335",
     govukYellow: "#FFBF47",
     white: "#ffffff",
-    cardGrey: "#f3f3f3"
+    cardGrey: "#f3f3f3",
+    black: "#000000"
```
Comment (sastels) [inline theme.js:38]: Are you using black in this PR or was it just since its a good thing to have?

File: __tests__/components/BB_test.js [hunk 2]
```diff
@@ -95,7 +95,7 @@ describe("BB", () => {
   });
 
   it("has a sortBy selector", () => {
-    expect(shallow_BB().find("#sortBySelector").length).toEqual(1);
+    expect(mounted_BB().find("#sortBySelector").length).toEqual(2);
```
Comment (sastels) [inline __tests__/components/BB_test.js:98]: why did this change to 2? mount vs shallow?

**Review by sastels — APPROVED** [review]
minor questions, looks sharp! 🦑

File: __tests__/components/BB_test.js (hunk 2 above)
Comment (smcmurtry) [reply __tests__/components/BB_test.js:98]: not sure - I chalked it up to the issue where enzyme sometimes returns multiple copies of the same thing.

File: theme.js (hunk 1 above)
Comment (smcmurtry) [reply theme.js:38]: I think I was using it at some point but no longer. But it's probably good to have in there.

**Pull request merged**